### DISCUSSION TITLE
Start Engine service

### DISCUSSION THREAD

**Discussion by TobiasLaross** [TobiasLaross/Hass-Lynk-Co#30] [General]
What are your findings regarding start engine service? My working theory is currently:
It starts the climate on the latest configuration you had in the car and the cable must be unplugged unlike start climate. Currently the engine will run for 15 minutes and that will be configurable in a future update between 1-15 minutes.

**Comment by SYSKOMM** [comment]
Good morning,
I cannot find the air conditioning control and the engine start control in HA.
THANKS

**Reply by TobiasLaross** [reply]
There is no option to change air conditioning, I think it will run on the latest settings from the car, similar to if you started the engine by pushing start in the car
